src/databricks/labs/ucx/contexts/workspace_cli.py

```diff
@@ -179,6 +180,10 @@ def iam_role_creation(self):
     def notebook_loader(self) -> NotebookLoader:
         return NotebookLoader()
 
+    @cached_property
```

Comment:
can you remove `HiveMetastoreFederation` from this PR? let's have it added in a separate one.

Reply:
Done